### PR TITLE
Everyone on support.

### PR DESCRIPTION
**Pull request by marbeann**
Policy for EOS.
Ref: https://github.com/niteoweb/operations/issues/81

### CODE REVIEW

File: everyone-on-support.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Prologue
+
+Everyone on Support is a regular activity wherein all full-time members are called upon to do a Support shift once every month (this includes customer and tech support). Trialists and part-time team members are excluded from this ritual.
```
Comment (dmurko) [inline everyone-on-support.md]: I wouldn't use "ritual" :)

File: everyone-on-support.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Prologue
```
Comment (dmurko) [inline everyone-on-support.md]: I would use Overview or Introduction, to keep with the standard wording of Handbook.

File: everyone-on-support.md (hunk 1 above)
Comment (dmurko) [inline everyone-on-support.md]: "Everyone On Support (EOS)..."

File: everyone-on-support.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+In the end, we are working for our customers, and having everyone involved in the conversation will help improve their experience and make them happy. And happy customers are happy to spend money with us.
+
+# Mechanics
+
+## I. Instructions
+
+* We use GrooveHQ for handling support emails. For EOS, we'll have one account which you'll need to personalize when you're on duty. Login credentials are on 1Password. Go to Settings and update the Name field.
+* Everyone has Staff access to the EBN app.
+* GrooveHQ integrates with Slack so we get a live stream of emails and responses. You can optionally join #support-emails channel.
+* You'll need to brush up on proper support decorum. See our Handbook.
```
Comment (dmurko) [inline everyone-on-support.md]: "See [Handling Support Requests](https://github.com/niteoweb/handbook/blob/master/support.md)."

File: everyone-on-support.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* All support documents can be found in Intra Easy Blog Networks folder. Make sure you go through them regularly and suggest improvement where needed.
```
Comment (dmurko) [inline everyone-on-support.md]: ...can be found in [Intra EBN folder](https://intra.niteoweb.com/projects/easy-blog-networks/support-documents)."

If Zupo says internal links are OK, then I'm OK.

File: everyone-on-support.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+## II. Frequency
+
+EOS will be once every month. 
+
+Since we're on Sprints, it's expected that Team Capacity will be affected and duly adjusted.
+
+## III. Schedule
+
+Each one will set their own schedule and keep with it. Set a date and update the calendar in Intra.
```
Comment (dmurko) [inline everyone-on-support.md]: "Set a date and [update the calendar in Intra](https://intra.niteoweb.com/operations/everyone-on-support/view)."

File: everyone-on-support.md (hunk 4 above)
Comment (zupo) [inline everyone-on-support.md]: I'm preping "permalinks" in Intra, which won't reveal anything about internal structure in Intra. Should be deployed in a few minutes, then you go to Actions -> Copy Permalink -> Right click -> Copy Address.

Will let you know when it is ready.

File: everyone-on-support.md (hunk 5 above)
Comment (zupo) [inline everyone-on-support.md]: We someone to be in charge of poking everyone to do so. @marbeann, wanna volunteer? And we can say everyone needs to set their date by day 10 of each quarter.

Comment (marbeann) [inline everyone-on-support.md]: Sure. 

However, `And we can say everyone needs to set their date by day 10 of each quarter.`, does this mean when set schedule, we're setting it for the whole quarter? Won't that backfire by having multiple postponements due to unforeseen reasons?

I'm fine with adding this to my monthly (1st day of the month) tasks, and pinging everyone to set their date for that month.

File: everyone-on-support.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# Introduction
+
+Everyone on Support (EOS) is a regular activity wherein all full-time members are called upon to do a Support shift once every month (this includes customer and tech support). Trialists and part-time team members are excluded from this.
+
+EOS is not designed to make everyone support or to be pulled from regular work to do support. The main reason is that a new set of eyes can help find issues and opportunities in current processes. A developer might see something that is not obvious to non-dev. A marketer might see a great opportunity that was dismissed till then.
```
Comment (marbeann) [inline everyone-on-support.md]: > make everyone support -> make everyone do support?

Used as Proper noun here, should be capitalized.

File: everyone-on-support.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+# Mechanics
+
+### I. Instructions
+
+* We use [GrooveHQ](https://niteoweb.groovehq.com/) for handling support emails. For EOS, we'll have one account which you'll need to personalize when you're on duty. Login credentials are on 1Password. Go to Settings and update the Name field.
+* Everyone has Staff access to the EBN app.
+* GrooveHQ integrates with Slack so we get a live stream of emails and responses. You can optionally join #support-emails channel.
+* You'll need to brush up on proper support decorum. See [Handling Support Requests](https://github.com/niteoweb/handbook/blob/master/support.md).
+* All support documents can be found in  [Intra EBN folder](https://intra.niteoweb.com/projects/easy-blog-networks/support-documents?_authenticator=dfadc77722eb429e0c160346cceba5b05d39702e). Make sure you go through them regularly and suggest improvements where needed.
```
Comment (zupo) [inline everyone-on-support.md]: @marbeann: did you do Actions -> Click for Permalink -> right click -> copy address?

Left-clicking does not work, yet, there's a bug with Javascript.

Comment (marbeann) [inline everyone-on-support.md]: @zupo Left-clicked. Updated with the correct link now for all Intra URLs.

**Comment by zupo** [comment]
I've squashed the commits together into one and added one commit of cleanup. Now merging.

**Pull request merged**